x-pack/test/api_integration/apis/endpoint/alerts/index.ts

```diff
@@ -73,8 +75,17 @@ export default function ({ getService }: FtrProviderContext) {
   describe('Endpoint alert API', () => {
     describe('when data is in elasticsearch', () => {
       before(async () => {
-        await esArchiver.load('endpoint/alerts/api_feature', { useCreate: true });
-        await esArchiver.load('endpoint/alerts/host_api_feature', { useCreate: true });
+        await indexHostsAndAlerts(
+          client,
+          'alerts-seed',
+          numberOfHosts,
+          1,
+          'metrics-endpoint.metadata-default-1',
+          'metrics-endpoint.policy-default-1',
```

Comment:
I must have missed that these tests were using policy stuff before. Could you add a `deletePolicyStream(getService)` call to the `after` to clean those up.

Reply:
I think we weren't using the policy stuff before, but this generates it all together in a nice way. I'll add the `deletePolicyStream(getService)`